tests/test_html_output.py

```diff
@@ -100,6 +132,41 @@ def check_step_not_collapsed(section, body):
     assert body.is_displayed()
 
 
+def check_section_elements_classes(step, is_failed=False):
+    assert step.get_section_title_class() == "sectionTitle"
+    step.click() # open section body
+    exp_status = "Failed" if is_failed else "Success"
+    assert step.get_section_status_class() == f"section{exp_status}Status"
+    step.click() # close section body
+
+
+def check_body_style(style):
+    body_style = parse_css_block(style, "body")
+    assert parse_css_property(body_style, "background-color") == "white"
+
+
+def check_css_class(style, name, exp_color, exp_font):
+    class_style = parse_css_block(style, f".{name}")
+    assert parse_css_property(class_style, "color") == exp_color
+    assert parse_css_property(class_style, "font-weight") == exp_font
+
+
+def parse_css_block(style, block_name):
+    return re.search(block_name + r" \{(.*?)\}", style, re.DOTALL).group(1)
+
+
+def parse_css_property(style, css_property):
+    return re.search(fr"{css_property}: (\w+)", style).group(1)
+
+
+def get_page_style(browser):
+    head = browser.find_element_by_tag_name("head")
+    assert head
+    style = head.find_element_by_tag_name("style")
+    assert style
+    return style.get_attribute("innerHTML")
+
+
 class TestElement(FirefoxWebElement):
```

Comment:
With this class name the pytest produces the following warning:

tests/test_html_output.py:103
  est_html_output.py:103: PytestCollectionWarning: cannot collect test class 'TestElement' because it has a __init__ constructor (from: tests/test_html_output.py)
    class TestElement(FirefoxWebElement):